### DISCUSSION TITLE
What 🦸🦹 does Ryan Reynolds play?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#853] [Galaxy Trivia] [answered]
**Bonus:** Name the 🎥🎥.

**Comment by peterbaccaro** [comment]
Ryan Reynolds plays **Deadpool**.

Movies:
- **Deadpool [2016]**
- **Deadpool 2 [2018]**
- Deadpool 3 [2024] (being released next year)

**Comment by Carol42** [comment, marked as answer]
Ryan Reynolds portrays two heroes 🦸‍♂️, Deadpool and Green Lantern. He assumes the role of Deadpool in multiple films 🎥, including X-Men Origins: Wolverine (2009), Deadpool (2016), Deadpool 2 (2018), and the upcoming Deadpool (2024). In addition, he portrays Green Lantern in the film titled Green Lantern (2011). It's worth mentioning that he also played the character Hannibal King in Blade: Trinity (2004), although he doesn't fit the traditional hero archetype.

**Reply by ParanoidUser** [reply]
Well done! 🔥
